www/go/core/jmap/EntityController.php

```diff
@@ -613,6 +629,48 @@ private function createEntitites($create, &$result) {
 
 			if ($entity->save()) {
 
+				// save address book
```

Comment:
I see this code is moved to `www/go/modules/community/filteraddressbook/Module.php`

So we should remove it from here, correct?